variables.tf

```diff
@@ -7,6 +7,20 @@ variable "required_apis" {
   }
 }
 
+variable "for_gcr" {
+  type        = bool
+  default     = false
+  description = "If set to true, configure the service-account for GCR integration"
+}
+
+# This variable is set to true by default so that we don't break existing
+# workflows from our Cloud Account modules, gcp-config and gcp-audit-log
+variable "for_compliance" {
+  type        = bool
+  default     = true
+  description = "If set to false, avoid configuring the service-account for Compliance integration"
```

Comment:
Question on the for description. "avoid configuring..." sounds like a warning rather than something definitive. Is that intentional?

Reply:
I can update it! Good call

Reply:
agreed with @dmurray-lacework